routing/ROUTING_V1_HTTP.md

```diff
@@ -54,10 +54,13 @@ Both read and write provider records have a minimal required schema as follows:
 
 Where:
 
-- `Protocol` is the multicodec name of the transfer protocol or an opaque string (for experimenting with novel protocols without a multicodec)
+- `Protocol` is a globally unique opaque string that represents the transfer protocol used by a content provider.
```

Comment:
`globally unique` and `opaque` are somewhat conflicting, aren't they? If it's opaque, how do we guarantee that it's globally unique? maybe i'm misunderstanding how opaque applies here, but maybe opaque is a "golang-esque" terminology that shouldn't be in the spec?

side note: I think we overuse the term "opaque"